README.md

```diff
@@ -120,14 +136,28 @@ If you would prefer to look at our most up-to-date work, please check out the fo
 
 ---
 
-## Local Development
+## Architectural Comment
 
-We've tried to make it as easy as possible to go from fork/clone to a running instance.
+Our demonstration application run in us-west2, while the Augur database that we use is in Columbus, MO. To improve user experience,
```

Comment:
Outside of Red Hat I dont think people will know what "us-west2" means

Reply:
Oh, that's an AWS zone. I'll clarify but that's a conventional AWS region that's well-known.

Reply:
updated!